tgstation.dme

```diff
@@ -216,6 +261,7 @@
 #include "code\datums\helper_datums\teleport.dm"
 #include "code\datums\helper_datums\topic_input.dm"
 #include "code\datums\spells\area_teleport.dm"
+#include "code\datums\spells\barnyard.dm"

```

Comment:
Down here is where your new file was added correctly, the above FILE_DIR is dream maker being dumb and can be removed.